### PR TITLE
fix: use correct spelling of JavaScript and TypeScript

### PR DESCRIPTION
**Pull request by NMinhNguyen**
Would be good to update the repo description also, but I don't have the rights to do so 😅  
image

### CODE REVIEW

**Comment by buttercubz** [comment]
@NMinhNguyen hacktoberfest?

**Comment by NMinhNguyen** [comment]
@buttercubz Feel free to label it as such (as it's only opt-in now) but inconsistent spelling of JS/TS does bother me a little so thought I'd contribute.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Start by creating an issue about your feature or bug! Then, create a PR and we'l
 
 **Why choose QuickJS over V8?**
 
-QuickJS is a small and embeddable Javascript engine but it lacks V8's JIT for fast JavaScript execution. Although, it doesn't mean you cannot use Elsa on backends and CPU intensive tasks.
+QuickJS is a small and embeddable JavaScript engine but it lacks V8's JIT for fast JavaScript execution. Although, it doesn't mean you cannot use Elsa on backends and CPU intensive tasks.
```
Comment (NMinhNguyen) [inline README.md:66]: Especially when it's inconsistent within the same sentence 😅

**Review by littledivy — APPROVED** [review]
LGTM. Every PR has an importance, I'll mark it for hacktoberfest :)

**Pull request merged**